platform-coop-toolkit/coop-map-directory-index
# Geocoding steps

## Description

This PR adds a `geolocation` step after the `contact_info` step in both the individual and organization profile flow.

## Steps to test

1. Initiate individual profile creation
2. Verify a map appears at the third step
3. Adjust the location or not
4. Continue to end of profile creation
5. Verify the same location appears in the `/admin` interface for that `User`
6. Initiate organization profile creation
7. Verify a map appears at the third step
8. Adjust the location or not
9. Continue to end of profile creation
10. For a change of pace to the homepage and verify that the new `Organization` appears where expected on the map

**Expected behavior:** 

## Additional information

Outstanding issues:
* there is a hack to pull `lng` & `lat` into hidden fields that are immediately copied to other hidden fields. Something is incorrect in the form `context`.
* I've done nothing to make the crosshairs redraw if the map dimensions change.
* the use of the Mapbox `light` style is a placeholder to differentiate from the homepage map.

## Related issues

I considered this part of #6 & #7 but the emphasis of those issues evolved. This deals with complete
